README.md

```diff
@@ -1,61 +1,143 @@
+- Scale seamlessly
+- Simplify maintenance
+- Monitor and optimize
+- Automate backups
+- Ensure data security
+
+If you'd like to get a complete understanding of the features offered by Percona Everest, click on this [link](https://percona.community/projects/everest/).
+
+## Documentation
+
+For comprehensive information about Percona Everest, see the [documentation](https://docs.percona.com/everest/index.html).
+
+# Installation
+
+Ready to try out Percona Everest? Check the [Quickstart install](https://docs.percona.com/everest/quickstart-guide/quick-install.html) section for easy-to-follow steps. 
 
 ## Prerequisites
 
-A Kubernetes cluster is available for public use, but we do not offer support for creating one.
+Before getting started with Percona Everest, do the following:
+
+1. Set up a Kubernetes cluster. 
+
+    Percona Everest assists with installing all the necessary operators and required packages, but does not deploy a Kubernetes cluster.
+
+    We recommend setting up Percona Everest on the [Amazon Elastic Kubernetes Service (EKS)](https://docs.percona.com/everest/quickstart-guide/eks.html) or [Google Kubernetes Engine (GKE)](https://docs.percona.com/everest/quickstart-guide/gke.html).
+
+2. Verify that you have access to the Kubernetes cluster that you want to use with Everest. By default, Everest uses the kubeconfig file available under `~/.kube/config`. 
+
+    To verify access to the Kubernetes cluster, run the following command:
+   
+    ```sh 
+    kubectl get nodes
+    ```
+
+## Install Percona Everest
+
+To install and provision Percona Everest to Kubernetes:
+
+
+1. Install the latest version of the Everest CLI by running the following commands:
+
+    **Linux and WSL**
+        
+    ```sh
+    curl -sSL -o everestctl-linux-amd64 https://github.com/percona/percona-everest-cli/releases/latest/download/everestctl-linux-amd64
+    sudo install -m 555 everestctl-linux-amd64 /usr/local/bin/everestctl
+    rm everestctl-linux-amd64
+    ```
+
+    **macOS (Apple Silicon)**
+
+    ```sh
+    curl -sSL -o everestctl-darwin-arm64 https://github.com/percona/percona-everest-cli/releases/latest/download/everestctl-darwin-arm64
+    sudo install -m 555 everestctl-darwin-arm64 /usr/local/bin/everestctl
+    rm everestctl-darwin-arm64
+    ```
+
+    **macOS (Intel CPU)**
+
+    ```sh
+    curl -sSL -o everestctl-darwin-amd64 https://github.com/percona/percona-everest-cli/releases/latest/download/everestctl-darwin-amd64
+    sudo install -m 555 everestctl-darwin-amd64 /usr/local/bin/everestctl
+    rm everestctl-darwin-amd64
+    ```
+
+2. Install Everest and provision the Kubernetes cluster using one of the following commands:
+
+    ```sh
+    everestctl install
+    ```
+
+    Enter the specific names for the namespaces you want Everest to manage, separating each name with a comma.
+
+    !!! note 
+        Make sure that you enter at least one namespace.
+
+    Alternatively, you can set multiple namepaces in the headless mode:
```

Comment:
Same here

Reply:
The formatting is correct. Let me check why its not rendering properly.